Dockerfile

```diff
@@ -136,6 +146,15 @@ RUN cd /nextstrain/auspice && npm update && npm install && npm run build && npm
 FROM python:3.7-slim-buster
 
 # Add system runtime deps
+# bzip2, gzip, xz-utils, zip, unzip, zstd: install compression tools
+# ca-certificates: ?
+# curl: [Dockerfile] for downloading binaries directly; may be used by workflows
+# jq: ?
```

Comment:
@corneliusroemer do you remember why you added `jq` in 8a6a29d906de9546c8b2c6be118bb95b206ba868? Is it just for usability in workflows?